src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -5685,19 +5676,21 @@ public void setEnforceNativeDeadlineForHints(boolean value)
         DatabaseDescriptor.setEnforceNativeDeadlineForHints(value);
     }
 
+    @Override
     public List<String> getTablesForKeyspace(String keyspace)
     {
         return Keyspace.open(keyspace).getColumnFamilyStores().stream().map(cfs -> cfs.name).collect(Collectors.toList());
     }
 
-    public List<String> mutateSSTableRepairedState(boolean repaired, boolean preview, String keyspace, List<String> tableNames) throws InvalidRequestException
+    @Override
+    public List<String> mutateSSTableRepairedState(boolean repaired, boolean preview, String keyspace, List<String> tableNames)
     {
         Map<String, ColumnFamilyStore> tables =  Keyspace.open(keyspace).getColumnFamilyStores()
                                                          .stream().collect(Collectors.toMap(c -> c.name, c -> c));
         for (String tableName : tableNames)
         {
             if (!tables.containsKey(tableName))
-                throw new InvalidRequestException("Table " + tableName + " does not exist in keyspace " + keyspace);
+                throw new RuntimeException("Table " + tableName + " does not exist in keyspace " + keyspace);
```

Comment:
Tiny nit: this causes SSTableRepairedAtTest.testMutateSSTableRepairedStateTableNotFound to fail since it expects an InvalidRequestException